tests/e2e/dpa_deployment_suite_test.go

```diff
@@ -1,815 +1,409 @@
-									Bucket: bucket,
-									Prefix: lib.VeleroPrefix,
-								},
-							},
-							Credential: &bslCredential,
-						},
-					},
-				},
-				Configuration: &oadpv1alpha1.ApplicationConfig{
-					Velero: &oadpv1alpha1.VeleroConfig{
-						PodConfig: &oadpv1alpha1.PodConfig{},
-						DefaultPlugins: []oadpv1alpha1.DefaultPlugin{
-							oadpv1alpha1.DefaultPluginOpenShift,
-							oadpv1alpha1.DefaultPluginAWS,
-						},
-					},
-				},
-			},
-			WantError: true,
-		}, fmt.Errorf("region for AWS backupstoragelocation not automatically discoverable. Please set the region in the backupstoragelocation config")),
-		ginkgov2.Entry("unsupportedOverrides should succeed", ginkgov2.Label("aws", "ibmcloud"), InstallCase{
-			Name:         "valid-unsupported-overrides",
-			BRestoreType: lib.RESTIC,
-			DpaSpec: &oadpv1alpha1.DataProtectionApplicationSpec{
-				BackupLocations: []oadpv1alpha1.BackupLocation{
-					{
-						Velero: &velero.BackupStorageLocationSpec{
-							Provider: providerFromDPA,
-							Config: map[string]string{
-								"profile":          bslConfig["profile"],
-								"region":           bslConfig["region"],
-								"s3ForcePathStyle": "true",
-							},
-							Default: true,
-							StorageType: velero.StorageType{
-								ObjectStorage: &velero.ObjectStorageLocation{
-									Bucket: bucket,
-									Prefix: lib.VeleroPrefix,
-								},
-							},
-							Credential: &bslCredential,
-						},
-					},
-				},
-				Configuration: &oadpv1alpha1.ApplicationConfig{
-					Velero: &oadpv1alpha1.VeleroConfig{
-						PodConfig: &oadpv1alpha1.PodConfig{},
-						DefaultPlugins: []oadpv1alpha1.DefaultPlugin{
-							oadpv1alpha1.DefaultPluginOpenShift,
-							oadpv1alpha1.DefaultPluginAWS,
-						},
+			}),
+		}),
+		ginkgo.Entry("DPA CR with kopia enabled with node selector", InstallCase{
+			DpaSpec: createTestDPASpec(TestDPASpec{
+				BSLSecretName:   bslSecretName,
+				EnableNodeAgent: true,
+				NodeAgentPodConfig: oadpv1alpha1.PodConfig{
+					NodeSelector: map[string]string{
+						"foo": "bar",
 					},
 				},
+			}),
+		}),
+		ginkgo.Entry("DPA CR with NoDefaultBackupLocation and with BackupImages false", InstallCase{
+			DpaSpec: createTestDPASpec(TestDPASpec{
+				BSLSecretName:           bslSecretName,
+				NoDefaultBackupLocation: true,
+				DoNotBackupImages:       true,
+			}),
+		}),
+		ginkgo.Entry("DPA CR with S3ForcePathStyle true", ginkgo.Label("aws"), InstallCase{
+			DpaSpec: createTestDPASpec(TestDPASpec{
+				BSLSecretName:    bslSecretName,
+				s3ForcePathStyle: true,
+			}),
+		}),
+		// TODO bug https://github.com/vmware-tanzu/velero/issues/8022
+		// Entry("DPA CR without Region, without S3ForcePathStyle and with BackupImages false", Label("aws"), InstallCase{
+		// 	DpaSpec: createTestDPASpec(TestDPASpec{
+		// 		BSLSecretName:      bslSecretName,
+		// 		NoRegion:           true,
+		// 		NoS3ForcePathStyle: true,
+		// 		DoNotBackupImages:  true,
+		// 	}),
+		// }),
```

Comment:
FYI You can also change to `PEntry` so it gets skipped.

Reply:
I do not like how ginkgo prints skipped tests, think it pollutes the screen too much

but I can change if wanted

Reply:
nah just fyi. resolving.